recipes/coverage-lcov/meta.yaml

```diff
@@ -0,0 +1,40 @@
+{% set name = "coverage-lcov" %}
+{% set version = "0.2.1" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 3de40d620cb13e43c746126be3154252d1296787a9f2a54b14683763cf9c0c46
+
+build:
+  skip: true  # [win]
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - pip
+    - python
+    - poetry
+  run:
+    - python
+    - toml
+    - click
+    - coverage
```

Comment:
Please add the versions here from https://github.com/TheCleric/coverage-lcov/blob/9f485275d3c784b34150d0f129d4ee442dce0c23/pyproject.toml#L56-L60